src/azure-cli/azure/cli/command_modules/role/_help.py

```diff
@@ -47,6 +47,26 @@
             "isEnabled": "true",
             "value": "approver"
         }]
+  - name: Create an application with optional claims
+    text: |
+        az ad app create --display-name mytestapp --optional-claims @manifest.json
+        ("manifest.json" contains the following content)
```

Comment:
how about some description on fields of manifest.json

Reply:
I checked the swagger file, the descriptions of `idToken` and `accessToken` are just same with the name. Do you mean that we need to clarify why we only support these two properties rather than `smalToken` ?

Reply:
not why, but clear description/guide/sample(either way ok) of each fields, what's name/source/essentail mean to help user write a correct manifest.json

Reply:
FYI, the docs are at [optionalClaims attribute](https://docs.microsoft.com/en-us/azure/active-directory/develop/reference-app-manifest#optionalclaims-attribute) and [OptionalClaim type](https://docs.microsoft.com/en-us/azure/active-directory/develop/active-directory-optional-claims#optionalclaim-type).

Reply:
> not why, but clear description/guide/sample(either way ok) of each fields, what's name/source/essentail mean to help user write a correct manifest.json

I add OptionalClaim properties doc link in the help message of `--optional-claims`. It's because I think it will be a little strange to put the description in the example part. Do you think it is OK?